tilescopethree/strategies/batch_strategies/__init__.py

```diff
@@ -11,4 +11,3 @@
                                           row_and_col_placements,
                                           row_placements)
 from .requirement_corroboration import requirement_corroboration
-from .targeted_cell_insertion import targeted_cell_insertion
```

Comment:
I've removed this strategy and the file that contained because it is the same strategy as all factor insertion.